src/menuItem/ServiceItem.js

```diff
@@ -27,8 +27,8 @@ class ServiceItem extends MenuItem {
   visible() {
     return super.visible() &&
       this.menu.state.started &&
-      !this.serviceState.disabled &&
-      this.serviceState !== null;
+      this.serviceState !== null &&
```

Comment:
the order of checks was incorrect: we where checking for nested field in serviceState before checking that serviceState itself was not null